todoapp_server/users/views.py

```diff
@@ -1,3 +1,21 @@
-from django.shortcuts import render
+from rest_framework import viewsets, mixins
+from .serializers import UserSerializer, User
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.renderers import TemplateHTMLRenderer, JSONRenderer
+from rest_framework.status import HTTP_200_OK
+from rest_framework.decorators import action
 
-# Create your views here.
+class UserViewSet(mixins.CreateModelMixin,
+                viewsets.GenericViewSet):
+
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+    @action(detail=False, 
+        url_path='me', 
+        permission_classes=[IsAuthenticated],
+        renderer_classes=[JSONRenderer]) 
```

Comment:
ovo ti nije potrebno, trebalo bi da ti je po default-u JSONRenderer, ako nije onda napravi da jeste tako da ne moras za svaki view da ovako stavljas

Reply:
Jeste po default-u JSONRenderer, ja sam skroz prevideo da mi to sada ne treba..